api/workers_test.go

```diff
@@ -755,6 +760,7 @@ var _ = Describe("Workers API", func() {
 			fakeWorker.TagsReturns([]string{"some-tag"})
 			fakeWorker.StateReturns(db.WorkerStateRunning)
 			fakeWorker.TeamNameReturns("some-team")
+			fakeWorker.EphemeralReturns(false)
```

Comment:
Might be better to have this return `true`, since it'll also be false in the response if we forget to set the field in the API.